### PR TITLE
ngfw:13516 - interface column changes

### PR DESCRIPTION
**Pull request by Shriram-Patil-Arista**
before :
image

after:
Screenshot from 2024-04-17 19-01-32

### CODE REVIEW

File: uvm/servlets/admin/config/network/view/Interfaces.js [hunk 1]
```diff
@@ -199,6 +199,7 @@ Ext.define('Ung.config.network.view.Interfaces', {
             sourceConfig: {
                 device: { displayName: 'Device'.t() },
                 macAddress: { displayName: 'MAC Address'.t() },
+                vendor: { displayName: 'Vendor'.t() },
```
Comment (cblaise-untangle) [inline uvm/servlets/admin/config/network/view/Interfaces.js:202]: Let's clarify the displayName as "MAC Vendor"

File: uvm/servlets/admin/config/network/view/Interfaces.js [hunk 2]
```diff
@@ -260,7 +261,13 @@ Ext.define('Ung.config.network.view.Interfaces', {
                 dataIndex: 'address',
                 width: Renderer.ipWidth,
                 flex: 1
-            }],
+            },{
+                header: 'Vendor'.t(),
```
Comment (cblaise-untangle) [inline uvm/servlets/admin/config/network/view/Interfaces.js:265]: Let's clarify as "MAC Vendor".  Or maybe do a grouping of MAC address/vendor like we do under Devices.

File: uvm/servlets/admin/config/network/view/Interfaces.js (hunk 1 above)
Comment (Shriram-Patil-Arista) [reply uvm/servlets/admin/config/network/view/Interfaces.js]: Done

File: uvm/servlets/admin/config/network/view/Interfaces.js (hunk 2 above)
Comment (Shriram-Patil-Arista) [reply uvm/servlets/admin/config/network/view/Interfaces.js]: Done

**Review by cblaise-untangle — APPROVED** [review]

**Pull request merged**